crimethinc/website
# Add plain text Markdown of Articles#show

# How does this pull request make you feel (in animated GIF format)?

![pretty good](https://media.giphy.com/media/u5SuJWBUCXPrjwN80e/giphy.gif)

# What are the relevant GitHub issues?

resolves #2422

# What does this pull request do?

This PR adds a `.markdown` format view for the  `articles#show` action.

<img width="1242" alt="Screen Shot 2022-02-19 at 7 30 17 PM" src="https://user-images.githubusercontent.com/4361/154827275-999f8a8d-eac1-47df-bd27-5978527f07cf.png">


# How should this be manually tested?

Go to any article page, add `.markdown` to the end of the URL.
You should see a plaintext markdown view with YAML front matter.

# Is there any background context you want to provide for reviewers?

Translations often come in as .docx/.odt files. If they came in as Markdown, that'd make it faster easier to get them published.
